modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase

```diff
@@ -487,6 +487,101 @@ definition {
 			totalPendingTaskCount = "1");
 	}
 
+	@description = "LRQA-55606 Automate LPS-106225: [Dashboard] Apply pagination on bulk reassign workflow instances"
+	@priority = "5"
+	test ApplyPaginationOnBulkReassign {
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "WorkflowMetrics#ApplyPaginationOnBulkReassign";
+
+		var siteName = TestCase.getSiteName(siteName = "${siteName}");
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow",
+			site = "${siteName}");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		for (var blogNumber : list "1,2,3,4,5,6") {
+			JSONBlog.addEntry(
+				entryContent = "Blogs Entry ${blogNumber} Content",
+				entryTitle = "Blogs Entry Title ${blogNumber}");
+		}
```

Comment:
Create a macro for this part of the code. Similar to `WorkflowMetrics.createAlternativeInstances`.